.taskcluster.yml

```diff
@@ -24,9 +24,15 @@ tasks:
           go get -t github.com/taskcluster/taskcluster-cli/... 
           && cd  /go/src/github.com/taskcluster/taskcluster-cli 
           && git fetch {{ event.head.repo.url }} {{ event.head.ref }} 
-          && git checkout {{ event.head.sha }} 
           && make 
-          && go test ./...
+          && go get -u github.com/alecthomas/gometalinter 
+          && gometalinter --install
+          && go get -u ./...
+          && git checkout {{ event.head.sha }} 
```

Comment:
Does that mean that we're running our tests against the latest versions of everything, rather than the vendored versions?  Or is this just updating gometalinter's own dependencies?